### ISSUE TITLE
Create translations field not working

### ISSUE THREAD

**Issue by Mrmiffo** [directus/directus#6070]
**The issue and what you expected to happen**
In a fresh install of Directus it's not possible to properly create a translations field. The languages and translations collections are properly created, but the field in the parent collection is not working as intended. It's not displayed as expected (with a link to the translations collection) in the Data model screen, and the interface is empty on the edit collection item screen.
![image](https://user-images.githubusercontent.com/8545443/120777756-eae0e080-c525-11eb-93ba-e2b41413a769.png)
![image](https://user-images.githubusercontent.com/8545443/120777841-0350fb00-c526-11eb-8072-9f30086d2464.png)

**Exact steps to reproduce this issue**
Installed directus using `npm install` (as I have an existing .env file) and `npx directus bootstrap`
Create a collection called test
Add a string field called name
Add a translations field using Auto-generate

**Your environment:**
    MySQL 8
    npx (bootstrap)
    Chrome

**Any other relevant information we might need to reproduce this issue**
[Dump20210604.zip](https://github.com/directus/directus/files/6597234/Dump20210604.zip)

**Comment by iksent** [comment]
Am I experiencing the same issue or not?

Directus 0.69 (There are link to translation fields):
<img width="867" alt="Снимок экрана 2021-07-11 в 13 54 59" src="https://user-images.githubusercontent.com/32956521/125189060-d5ab5000-e24f-11eb-842e-c761b8662208.png">

Directus 0.83 (No link)
![2021-07-11_13-55-26](https://user-images.githubusercontent.com/32956521/125189082-ea87e380-e24f-11eb-80d2-e152858ebab9.jpg)

**Comment by rijkvanzanten** [comment]
@iksent That's correct and on purpose. We decided to standardize the design between fields vs groups. We're currently investigating adding a similar relational field style for all relationship types 👍🏻
